### PR TITLE
Add arm job config

### PR DESCRIPTION
**Pull request by lboeman**
For deploying https://github.com/SolarArbiter/solarforecastarbiter-core/pull/390.

I wasn't sure how each job was set up. Does the arm sub directory here need it's own var file with the api user and key and a schedule etc?

### CODE REVIEW

**Comment by lboeman** [comment]
@alorenzo175 this PR adds vars for the user/apikey in ssm for the ARM data to the development configs. Is there anything else I need to set up here to get the cron jobs going?

**Comment by alorenzo175** [comment]
You need the vars file in the new arm directory. And might as well the directory in the production folder as well

**Comment by alorenzo175** [comment]
oo actually with https://github.com/SolarArbiter/solarforecastarbiter-core/pull/369, we need to adjust the command. We should remove the start time from the command and the vars files, and add ``--end`` before the end time

File: development/sfa-core/jobs/arm/vars [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace: development
+image_tag: latest
+network: doe arm
+schedule: '0 1 * * *'
```
Comment (alorenzo175) [inline development/sfa-core/jobs/arm/vars:4]:
```suggestion
schedule: '15 2 * * *'
```

to run after SRML

File: development/sfa-core/jobs/arm/vars [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+base_url: https://dev-api.solarforecastarbiter.org
+start: '$today 0 - 2 day'
+end: '$today 1'
```
Comment (alorenzo175) [inline development/sfa-core/jobs/arm/vars:7]:
```suggestion
end: '$today 2'
```

File: production/sfa-core/jobs/arm/vars [hunk 3]
```diff
@@ -0,0 +1,7 @@
+namespace: production
+image_tag: stable
+network: doe arm
+schedule: '0 1 * * *'
```
Comment (alorenzo175) [inline production/sfa-core/jobs/arm/vars:4]:
```suggestion
schedule: '15 2 * * *'
```

File: production/sfa-core/jobs/arm/vars [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+base_url: https://api.solarforecastarbiter.org
+start: '$today 0 - 2 day'
+end: '$today 1'
```
Comment (alorenzo175) [inline production/sfa-core/jobs/arm/vars:7]:
```suggestion
end: '$today 2'
```

**Comment by lboeman** [comment]
Oh, sweet! I'll remove start and add `--end` for all the jobs.

**Pull request merged**